### PR TITLE
Changes in GUI and PriosinerABC

### PR DESCRIPTION
**Pull request by MickyHoltzman**

### CODE REVIEW

**Comment by MickyHoltzman** [comment]
Wait before you approve, I am not done, yet :-)
But, please take a look at the experiment manager, We had many conflicts which I resolved manually. Please make sure I did not screw up anything. Your changes had to do with moving the logging points around,

**Review by Anna4142 — DISMISSED** [review]

**Review by Anna4142 — COMMENTED** [review]
please ignore the approve.i think the logging part is not correct in this branch.can u compare it to the branch CorrectingDataLoggingInExpManager to see the diferences.I am sure that branch has the correct logging

File: Experiment_Launcher_code/ExperimentManager.py [hunk 1]
```diff
@@ -120,7 +112,6 @@ def StateActivity(self, state, mouse1, mouse2):
             self.cc_cnt += 1
             mouse1.DeliverReward(Locations.Cooperate, self.reward_time)
             mouse2.DeliverReward(Locations.Cooperate, self.reward_time)
-            self.time_to_make_decision = time.time() - self.time_start
```
Comment (Anna4142) [inline Experiment_Launcher_code/ExperimentManager.py]: why did u delete this?

File: Experiment_Launcher_code/ExperimentManager.py [hunk 2]
```diff
@@ -134,7 +125,6 @@ def StateActivity(self, state, mouse1, mouse2):
             self.cd_cnt += 1
             mouse1.DeliverReward(Locations.Defect, self.sucker_time)
             mouse2.DeliverReward(Locations.Cooperate, self.temptation_time)
-            self.time_to_make_decision = time.time() - self.time_start
```
Comment (Anna4142) [inline Experiment_Launcher_code/ExperimentManager.py]: same here

File: Experiment_Launcher_code/ExperimentManager.py [hunk 3]
```diff
@@ -148,7 +138,6 @@ def StateActivity(self, state, mouse1, mouse2):
             self.dc_cnt += 1
             mouse1.DeliverReward(Locations.Cooperate, self.temptation_time)
             mouse2.DeliverReward(Locations.Defect, self.sucker_time)
-            self.time_to_make_decision = time.time() - self.time_start
```
Comment (Anna4142) [inline Experiment_Launcher_code/ExperimentManager.py]: and here.you havent deleted it in the M1DM2D state.I will prefer if you leave it be in these states despite the redundency

**Review by Anna4142 — COMMENTED** [review]
i have made comments in the exp manager

File: Experiment_Launcher_code/ExperimentManager.py (hunk 1 above)
Comment (MickyHoltzman) [reply Experiment_Launcher_code/ExperimentManager.py]: Fixed

File: Experiment_Launcher_code/ExperimentManager.py (hunk 2 above)
Comment (MickyHoltzman) [reply Experiment_Launcher_code/ExperimentManager.py]: fixed

File: Experiment_Launcher_code/ExperimentManager.py (hunk 3 above)
Comment (MickyHoltzman) [reply Experiment_Launcher_code/ExperimentManager.py]: fixed

**Review by Anna4142 — APPROVED** [review]

**Pull request merged**